lib/ui/dashboard.js

```diff
@@ -514,11 +514,53 @@ class Dashboard {
       return { promise, cancellation: request.cancellation };
     }
 
+    const validElements = {
+      pods: "Pods",
+      logs: "Logs"
+    }
+
     this.render = function () {
       screen.append(pods_table);
       screen.append(resources);
       screen.append(pod_log);
-      pods_table.focus();
+
+      screen.key(['o', 'p'], function (_, key) {
+      
+        // get array of values from `validNodes` dictionary.
+        const validElementValues = Object.values(validElements);
+        const elements = screen.children;
+
+        let elementName;
+
+        // set target element label from `validNodes` dictionary.
+        if (key.name === 'o') elementName = validElements.logs;
+        if (key.name === 'p') elementName = validElements.pods;
+
+        // blur all valid elements.
+        elements.forEach(element => {
+          const label = element.options.label;
+          if (!validElementValues.includes(label)) return;
+          handleElementBlur(element);
+        });
+
+        // find target element.
+        const target = elements.find(element => element.options.label === elementName);
+        handleElementFocus(target);
+      });
+
+      handleElementFocus(pods_table);
+    }
+
+    // blur element & remove highlight from border.
+    function handleElementBlur(element) {
```

Comment:
May I suggest to use a bullet symbol in the title of the focused widget as https://github.com/wagoodman/dive does. It will help if we want to implement proper theming eventually.